ntc_templates/templates/alcatel_aos_show_interfaces_port.textfsm

```diff
@@ -0,0 +1,18 @@
+Value PORT ((\d+\/?)+)
+Value PERMANENT_SHUTDOWN (\*?)
+Value ADMIN_STATUS (\S+)
+Value LINK_STATUS (\S+)
+Value VIOLATIONS (\S+)
+Value RECOVERY_TIME (\d+)
+Value RECOVERY_MAX (\d+)
+Value WTR_RUNNING (\#?)
+Value WTR_TIME (\d+)
+Value ALIAS (.*)
+
+Start
+  ^\s*-+\+ -> Table
```

Comment:
```suggestion
  ^\s*-+\+ -> Table
  ^Legends: WTR - Wait To Restore
  ^\s+#
  ^\s+*
  ^\s*Slot/\s+Admin\s+Link\s+Violations\s+Recovery\s+Recovery\s+WTR\s+Alias\s*$$
  ^\s*Port\s+Status\s+Status\s+Time\s+Max\s+(sec)\s*$$
```

I like to match the table headers to ensure we are getting the headers we expect

Reply:
Looks good to me!